ILRepack/RepackOptions.cs

```diff
@@ -195,6 +198,32 @@ void Parse()
                         throw new InvalidTargetKindException("Invalid target: \"" + targetKind + "\".");
                 }
             }
+
+            var readDebugSymAs  = cmd.Option("readDebugSymbolAs");
```

Comment:
Can Cecil not detect the pdb type? Also, the default maybe should be the full pdb as that is what it used to be? Or rather, if cecil can detect the pdb type then the defaults should be the read type?